From c84fb76ee83e6bf69abfb69d77b3d1400b88fcdb Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Fri, 15 May 2015 17:14:36 -0700
Subject: [PATCH 518/793] drm: xilinx: drv: Add set_busid() callback

This function is required by some clients (ex, X11 armsoc), and
the Xilinx DRM was missing. Add the callback to Xilinx DRM.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/gpu/drm/xilinx/xilinx_drm_drv.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/gpu/drm/xilinx/xilinx_drm_drv.c b/drivers/gpu/drm/xilinx/xilinx_drm_drv.c
index 7ed987a..a515d66 100644
--- a/drivers/gpu/drm/xilinx/xilinx_drm_drv.c
+++ b/drivers/gpu/drm/xilinx/xilinx_drm_drv.c
@@ -380,6 +380,7 @@ static struct drm_driver xilinx_drm_driver = {
 	.unload				= xilinx_drm_unload,
 	.preclose			= xilinx_drm_preclose,
 	.lastclose			= xilinx_drm_lastclose,
+	.set_busid			= drm_platform_set_busid,
 
 	.get_vblank_counter		= drm_vblank_count,
 	.enable_vblank			= xilinx_drm_enable_vblank,
-- 
1.7.5.4

